test/modules/engin/pkg_module_res.future

```diff
@@ -0,0 +1,6 @@
+bug: function resolution on package modules
+
+Lydia's comment on the issue:
```

Comment:
Let's be a little more descriptive here, in case a significant amount of time goes by before this is returned to.  Namely, we want to call out that what is broken is *explicit naming* of functions defined within a package module.  Line 7 works just fine, and line 8 is broken in all cases where a function in a standard module is explicitly named, even those without use statements.  Note that a longer description should be separate from the "bug:" header.

It might be helpful to mention the other modules you tried to access in this way, so someone approaching the problem knows as much information as possible.